packages/web/partials/layout.hbs

```diff
@@ -28,7 +28,7 @@
         padding: 0.25rem;
       }
 
-      .example-CheckboxField-grid, .example-TextField-grid {
+      .example-CheckboxField-grid, .example-RadioField-grid, .example-TextField-grid {
```

Comment:
Maybe the time has come to upgrade it to `.example-grid`? 🙂

Reply:
from my point of view now useless, but feel free to refactor it later if you have time for it..

Reply:
Actually I tweaked it during #424.